projects/plugins/jetpack/modules/contact-form/js/grunion-admin.js

```diff
@@ -192,6 +193,11 @@ jQuery( function ( $ ) {
 			selected.push( parseInt( $( this ).attr( 'value' ), 10 ) );
 		} );
 
+		var errorMessage =
+			( window.exportParameters && window.exportParameters.exportError ) ||
+			'There was an error exporting your results';
```

Comment:
Can we make this translatable as well?

Reply:
This is translatable through the newly introduced `localize_script` you suggested: https://github.com/Automattic/jetpack/pull/27821/files#diff-a703f8594def3850d0961898f2de0d39990b74db630650d66d8b723f66862d54R1169

The only reason I've added a fallback fixed to english is in case the script fails for some reason (so we don't get a null there?)

Reply:
Oh okay. That's not really something I would worry about, but it doesn't hurt to leave it here. 👍